### PR TITLE
Minor documentation updates

### PR DESCRIPTION
**Pull request by ginach**
@rgregg, @aclev

### CODE REVIEW

**Comment by aclev** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -62,7 +64,7 @@ The following sample applications are also available:
 * [OneDrive API Browser](samples/OneDriveApiBrowser) - Windows Forms app
 * [OneDrive Photo Browser](samples/OneDrivePhotoBrowser) - Windows Universal app
 
-To run the OneDrivePhotoBrowser sample app your machine will need to be configured for [UWP app development](https://msdn.microsoft.com/en-us/library/windows/apps/dn609832.aspx).
+To run the OneDrivePhotoBrowser sample app your machine will need to be configured for [UWP app development](https://msdn.microsoft.com/en-us/library/windows/apps/dn609832.aspx) and be associated with the Windows Store.
```
Comment (rgregg) [inline README.md]: Can you edit this to be a little more clear? I read this as "your machine needs to be associated with the Windows Store" and I think you meant the "project needs to be associated with the Windows Store".

Comment (ginach) [inline README.md]: Updated

**Comment by rgregg** [comment]
Ship it!

**Pull request merged**